src/main/kotlin/com/example/noswimmingserver/domain/reading_journal/service/QueryReadingJournalDetailService.kt

```diff
@@ -0,0 +1,37 @@
+package com.example.noswimmingserver.domain.reading_journal.service
+
+import com.example.noswimmingserver.domain.book.facade.BookFacade
+import com.example.noswimmingserver.domain.reading_journal.facade.ReadingJournalFacade
+import com.example.noswimmingserver.domain.reading_journal.presentation.dto.response.QueryJournalDetailResponse
+import com.example.noswimmingserver.domain.teacher.facade.TeacherFacade
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class QueryReadingJournalDetailService(
+    private val readingJournalFacade: ReadingJournalFacade,
+    private val bookFacade: BookFacade,
+    private val teacherFacade: TeacherFacade,
+) {
+
+    @Transactional(readOnly = true)
+    fun execute(readingJournalId: Long): QueryJournalDetailResponse {
+
```

Comment:
🚫 **[ktlint]** <no-empty-first-line-in-method-block> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First line in a method block should not be empty